Code/Mantid/docs/source/algorithms/ConvertToReflectometryQ-v1.rst

```diff
@@ -23,6 +23,17 @@ workspace does not contain this value, or if you wish to override this
 value you can do so by providing your own *IncidentTheta* property and
 enabling *OverrideIncidentTheta*.
 
+The algorithm also has the option to create a table of 'DumpVertexes' that resemble
+the transformation before NormalisedPolygon has been used to rebin.
+The table of DumpedVertexes will have uniform bin widths. whereas the table created after
```

Comment:
There is only one TableWorkspace generated. And that's the one used for debugging purposes. Remove the section about uniform bin widths. (I can explain further if needed)